dev/ci/annotate.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+cd "$(dirname "${BASH_SOURCE[0]}")/../../"
+set -ex
```

Comment:
@asdine we forgot to remove that `x`, it's going to be noisy otherwise. 
```suggestion
set -e
```